### ISSUE TITLE
an error occurred during build after the latest theme update

### ISSUE THREAD

**Issue by Gictorbit** [zzossig/hugo-theme-zzo#197]
I updated the theme and after that, I couldn't build my site because of an error
```
hugo server
```
```
Built in 5051 ms
Error: Error building site: render of "404" failed: execute of template failed: template: 404.html:31:15: executing "404.html" at <partial "footer/site-footer" .>: error calling partial: execute of template failed: template: partials/footer/site-footer.html:3:7: executing "partials/footer/site-footer.html" at <partial "footer/links-social.html" .>: error calling partial: execute of template failed: template: partials/footer/links-social.html:11:7: executing "partials/footer/links-social.html" at <partial "footer/links-feed.html" .>: error calling partial: "/mnt/MyFile/PROJECT/gictor.ir/themes/zzo/layouts/partials/footer/links-feed.html:5:33": execute of template failed: template: partials/footer/links-feed.html:5:33: executing "partials/footer/links-feed.html" at <.CurrentSection.FirstSection.Permalink>: nil pointer evaluating page.Page.FirstSection
```

**Comment by zzossig** [comment]
Did you update the Hugo version to `0.65.2`?

**Comment by Gictorbit** [comment]
Hmm, my Hugo version is `v0.65.0`. Its newest version of Hugo that exists in my package manager.
I'm using Arch Linux and since it's `rolling release` there is no version `65.2` of Hugo even in the `testing-repository` of the package manager I think we should wait until the new version of Hugo be available in package manager repositories

**Comment by NghiaMaster** [comment]
Same issue after update theme :((
![Screenshot-20200222-155614.png](https://i.postimg.cc/D0c3tBry/Screenshot-20200222-155614.png)](https://postimg.cc/XBXP5K1T)

**Comment by Archaeoraptor** [comment]
same problem on Manjaro， and I build the latest Hugo (0.65.2) by myself, it works

**Comment by Archaeoraptor** [comment]
> Did you update the Hugo version to `0.65.2`?
> I suspect you didn't update the zzo theme yet.
> And this issue related #195

Thanks for your beautiful work，but the Hugo projects and theme zzo changes nightly，when update，error may occur.
Could you consider to publish a release or pre-release for theme zzo？So we can update it with less problem.

**Comment by zzossig** [comment]
@Gictorbit @Archaeoraptor  Hey guys, Sorry for the confusion.
>Did you update the Hugo version to 0.65.2

This is not what I meant you have to update the Hugo version to 0.65.2.
Actually, I thought that this had been fixed no matter what versions use.

I fixed just now so that you don't need to update the version.

**Comment by Gictorbit** [comment]
hi thanks
I updated the theme and error occurred again with `hugo` and `hugo server`
```
Total in 28920 ms
Error: Error building site: render of "404" failed: execute of template failed: template: 404.html:31:15: executing "404.html" at <partial "footer/site-footer" .>: error calling partial: execute of template failed: template: partials/footer/site-footer.html:3:7: executing "partials/footer/site-footer.html" at <partial "footer/links-social.html" .>: error calling partial: execute of template failed: template: partials/footer/links-social.html:11:7: executing "partials/footer/links-social.html" at <partial "footer/links-feed.html" .>: error calling partial: "/mnt/MyFile/PROJECT/gictor.ir/themes/zzo/layouts/partials/footer/links-feed.html:5:118": execute of template failed: template: partials/footer/links-feed.html:5:118: executing "partials/footer/links-feed.html" at <.FirstSection.Permalink>: error calling FirstSection: runtime error: invalid memory address or nil pointer dereference
```

**Comment by zzossig** [comment]
I think you didn't update the theme
![주석 2020-02-23 130206](https://user-images.githubusercontent.com/52706977/75103352-dfbdb600-563c-11ea-9529-86c87d44232c.png)

![주석 2020-02-23 130245](https://user-images.githubusercontent.com/52706977/75103335-d3395d80-563c-11ea-8a1a-dfcaa5783d68.png)

**Comment by Gictorbit** [comment]
actually I didn't push the latest commit
I pushed the latest commit now you can check it

**Comment by zzossig** [comment]
Still doesn't work?

**Comment by Gictorbit** [comment]
> hi thanks
> I updated the theme and error occurred again with `hugo` and `hugo server`
> 
> ```
> Total in 28920 ms
> Error: Error building site: render of "404" failed: execute of template failed: template: 404.html:31:15: executing "404.html" at <partial "footer/site-footer" .>: error calling partial: execute of template failed: template: partials/footer/site-footer.html:3:7: executing "partials/footer/site-footer.html" at <partial "footer/links-social.html" .>: error calling partial: execute of template failed: template: partials/footer/links-social.html:11:7: executing "partials/footer/links-social.html" at <partial "footer/links-feed.html" .>: error calling partial: "/mnt/MyFile/PROJECT/gictor.ir/themes/zzo/layouts/partials/footer/links-feed.html:5:118": execute of template failed: template: partials/footer/links-feed.html:5:118: executing "partials/footer/links-feed.html" at <.FirstSection.Permalink>: error calling FirstSection: runtime error: invalid memory address or nil pointer dereference
> ```

as I said before It doesn't work

**Comment by Gictorbit** [comment]
I checked Arch package manager and Found a new version of hugo is available recently, so I updated hugo to `v0.65.2` and problem solved

**Comment by zzossig** [comment]
Actually, the Hugo `0.65.2` version is a buggy version.
Maybe tomorrow(Monday) new version of Hugo will be released.

**Comment by Gictorbit** [comment]
Aha so we will upgrade it

**Comment by odunayo12** [comment]
Hi, I am encountering the same problem. I started with this [Version ](https://github.com/zzossig/hugo-theme-zzo/tree/27bb4753a955717af796f37d9b07bd583fd2a49f) of the theme as of Nov 2. and `hugo 0.78.0/extended version` and everything was working well until 3 days ago. 

Following the recommendations above, I updated the theme to its latest [version](https://github.com/zzossig/hugo-theme-zzo/tree/9a5a692aabea0609387d3d6b11347011a526a0a0)  as well as `hugo` to `0.78.2/extended`

See the error log below running the `hugo server` command. 
![image](https://user-images.githubusercontent.com/19196838/99181420-b98df800-272e-11eb-88bb-124baafcb932.png)
